File: java/Java programming 1/src/session6/Circle.java
package session6;

public class Circle {
    private double radius;
    private String color;

    public Circle(double radius, String color){
        this.radius = radius;
        this.color = color;
    }

    public double getRadius() {
        return radius;
    }

    public String getColor() {
        return color;
    }
    public double getArea(){
        return Math.PI * radius * radius;
    }

    @Override
    public String toString() {
        return "Circle{" +
                "radius='" + radius + '\'' +
                ", color='" + color + '\'' +
                '}';
    }
}

File: C#/.Net/Lab4/Ex1/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Ex1
{
    public class Circle : GeometricObject
    {
        private double radius;
        public Circle(double x){
            this.radius = x;
        }
        public Circle(double x, string c, double w) : base(c,w){
            this.radius = x;
        }
        public override string ToString()
        {
            return"Circle has: radius is " + radius + ", color is " + PColor + ", weight is " + Pweight;
        }
        public override double findArea()
        {
            return Math.PI * radius*radius;
        }

        public override double findPerimeter()
        {
            return 2 * Math.PI * radius;
        }
    }
}